node/resolvers/search/index.ts

```diff
@@ -665,6 +668,7 @@ export const queries = {
       sort: convertOrderBy(args.orderBy),
       sellers,
       hideUnavailableItems,
+      allowRedirect,
```

Comment:
```suggestion
      options,
```
You can just pass only the `options` object here and extract it into the `productSearch` function, so when we add more properties to `options` we can create a function there that extracts everything at once. what do you think?

Reply:
Yeah, thats cool 👯